charts/cluster-api-provider-metal3/templates/apps_v1_deployment_ipam-controller-manager.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    cluster.x-k8s.io/provider: infrastructure-metal3
+    control-plane: controller-manager
+    controller-tools.k8s.io: "1.0"
+  name: ipam-controller-manager
+  namespace: capm3-system
+spec:
+  selector:
+    matchLabels:
+      cluster.x-k8s.io/provider: infrastructure-metal3
+      control-plane: controller-manager
+      controller-tools.k8s.io: "1.0"
+  template:
+    metadata:
+      labels:
+        cluster.x-k8s.io/provider: infrastructure-metal3
+        control-plane: controller-manager
+        controller-tools.k8s.io: "1.0"
+    spec:
+      containers:
+      - args:
+        - --webhook-port=9443
+        - --diagnostics-address=:8443
+        - --insecure-diagnostics=false
+        - --tls-min-version=VersionTLS13
+        command:
+        - /manager
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        image: quay.io/metal3-io/ip-address-manager:v1.9.2
```

Comment:
I believe this should be in the chart value.yaml ?

Reply:
I wasn't sure about this one.

I asked the metal team if this was being build for downstream but didn't get a response. I'll take another look

Reply:
Right now moving this to the values file will force the tag to `latest`.

I'm questioning the usefulness of the line that is trying to force the tag to a particular value for every image in every chart ... Maybe we should leave that to the chart author?

What do you think?